src/ensembl/src/content/app/browser/browser-bar/BrowserBar.tsx

```diff
@@ -45,32 +37,19 @@ export const BrowserBar = (props: BrowserBarProps) => {
       <div className={styles.browserResetWrapper}>
         <BrowserReset />
       </div>
-      <BrowserInfo
-        ensObject={props.ensObject}
-        isDrawerOpened={props.isDrawerOpened}
-      />
+      {props.ensObject ? (
+        <FeatureSummaryStrip
+          ensObject={props.ensObject}
+          isGhosted={props.isDrawerOpened}
+        />
+      ) : null}
```

Comment:
sorry, here the `null` is unnecessary; we can use the `&&` operator instead of the ternary

```
      { props.ensObject && (
        <FeatureSummaryStrip
          ensObject={props.ensObject}
          isGhosted={props.isDrawerOpened}
        />
      }
```

using `nulls` is
- absolutely important when you do not want the component to render anything at all (react components can't return undefined, but can return null)
- useful when the value you are testing for can be 0, e.g.:

```
{ things.length ? <SomeComponent /> : null }
```

because if we write `{ things.length && <SomeComponent />`, then the component will render `0` if the list of `things` is empty.

Otherwise, when we are just checking whether or not to include something in the JSX depending on a certain condition, it's more customary to use the `&&` operator rather than a ternary.

Reply:
Thanks Andrey. From what i understand we use null when we do not want the component to render anything, is that the same as returning blank? Because thats what i want the component to do. 
Also if i use the ternary can you please check if i need to change my test when checking if the component FeatureSummaryStrip is present.

Reply:
I'd argue that it is always safe to use the ternary variant as then we wouldn't need to bother what the checked condition would result in i.e. `0` or a valid string.

Reply:
I'd argue that it is always safe to use the ternary variant as then we wouldn't need to bother what the checked condition would result in i.e. `0` or a valid string.

Reply:
Let's consider several hypothetical scenarios:

```
const Foo = () => {
  return x ? <div>Hello</div> : null;
}
```
in this case, we have to use a ternary with `null`, because if a component should not render anything at all, it must return `null`. Returning `undefined` or `false` is not allowed.

Now compare this with the following:

```
const Foo = () => {
  return (
    <div>
      <p>Hello</p>
      { x &&
        <p>world</p>
      }
    </div>
  );
}
```
in this example, the component will always return a JSX Element (div), which may or may not contain the part "world". In this situation, when we are using a condition to check whether to include a component or not, we prefer the `&&` operator to the ternary syntax.

Now the last example:

```
const Foo = () => {
  return (
    <div>
      <p>Hello</p>
      { x
        ? <p>world</p>
        : <p>universe</p>
      }
    </div>
  );
}
```
Here, the component is always returning a JSX element, but we are using a ternary to decide which of the two possible children to render inside of this component.

Your case is the same as example 2 above.

Reply:
Touche. I didn't notice this was in the middle of the JSX code.